src/_layouts/default.erb

```diff
@@ -30,6 +30,9 @@
               </div>
             </article>
           </main>
+          <div class = "hidden sidebar text-xs p-4 2xl:flex flex-col">
```

Comment:
I'm not sure what's going on here, but it looks like you have two closing `</div>`s. Please check your indentation too.